tests/InputFiles/CurvedScalarWave/WorldtubeKerrSchild.yaml

```diff
@@ -59,6 +59,7 @@ DomainCreator:
         ExciseWithBoundaryCondition: Worldtube
       UseLogarithmicMap: false
     CenterOfMassOffset: [0., 0.]
+    CubeLength: Auto
```

Comment:
What do you think about making this `CubeScale: 1` instead so we can control the cube length relative to the separation, @AlexCarpenter46 @knelli2 ?

Reply:
I'd like to do whatever is more intuitive for a user who isn't super familiar with our BCO domains. I think my vote for which is clearer would be `CubeLength` because it's exactly what it says it is vs `CubeScale` where you have to know *a priori* that the scale is with respect to the separation. (of course you could write this in the docs, but I was going based off the option name)

Reply:
I had a version with that, but after discussing with @knelli2 we agreed this might be more intuitive and less easily confused with other things in the BCO that use the word Scale

Reply:
Just a drive-by opinion, so don't let me block useful progress, but my vote would be for a "scale"-type option, because then there would be no need for an "Auto" value.  Having fewer default-like things is better.

Reply:
@nilsvu and I decided to go with `CubeScale`. @AlexCarpenter46 could you please switch this?

Reply:
Will do!